### DISCUSSION TITLE
Using jumbotron

### DISCUSSION THREAD

**Discussion by chopinesque** [bootscore/bootscore#893] [Q&A] [answered]
I had this in my old bootstap theme, with a background color, how can I produce an equivalent effect with resources in this theme?

```
<div class="jumbotron">
<div class="container">
<h2><span style="color: #337ab7;">HEADER</span></h2>
<p>Content.</p>
<a class="btn btn-primary btn-lg" role="button" href="#">Button»</a></div>
</div>
```

**Comment by crftwrk** [comment, marked as answer]
Jumbotron has been removed in Bootstrap 5. You can create own Jumbotrons using utilities like in the [example page](https://getbootstrap.com/docs/5.3/examples/jumbotrons/). 
```html
<div class="bg-body-tertiary rounded-4 p-5">
  <div class="container">
    <h2><span style="color: #337ab7;">HEADER</span></h2>
    <p>Content.</p>
    <a class="btn btn-primary btn-lg" role="button" href="#">Button»</a>
  </div>
</div>
```

You can use prebuilt Jumbotron block pattern as well https://bootscore.me/documentation/block-patterns/#jumbotron

**Comment by chopinesque** [comment]
Got it, how can I control background colour?

**Reply by crftwrk** [reply]
In your case `bg-body-tertiary` adds the light gray background. Please read the Bootstrap docs for [bg](https://getbootstrap.com/docs/5.3/utilities/background/) and [text](https://getbootstrap.com/docs/5.3/utilities/colors/) color utilities classes.
